beacon.yaml

```
@@ -561,10 +562,12 @@ components:
             3986 format).
           example: 'http://example.org/wiki/Main_Page'
         info:
-          description: 'Additional structured metadata, key-value pairs.'
-          type: array
-          items:
-            $ref: '#/components/schemas/KeyValuePair'
+          description: 'Additional unspecified metadata about the dataset.'
+          type: object
+          example:
+            fileTypes: ['.bam', '.vcf']
```

To be consistent, we should also use generic examples:
```suggestion
            additionalInfoKey1: [ additionalInfoValue1, additionalInfoValue2]
```